src/Constants.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+abstract contract Constants {
+  // Addresses
+  // Defender
+  address internal constant ETHEREUM_DEFENDER_ADDRESS = 0xA2A8BccD38138f1169ADdb0f3df9236a3CCCd753;
+  address internal constant OPTIMISM_DEFENDER_ADDRESS = 0xCeA11E14067697C085e1142afd2540b23f18304D;
+
+  // MessageExecutor
+  address internal constant ERC5164_EXECUTOR_OPTIMISM = 0x890a87E71E731342a6d10e7628bd1F0733ce3296;
+
+  // Multisigs
+  address internal constant EXECUTIVE_TEAM_OPTIMISM_ADDRESS =
+    0x8d352083F7094dc51Cd7dA8c5C0985AD6e149629;
+
+  // Chain IDs
+  uint256 internal constant ETHEREUM_CHAIN_ID = 1;
+  uint256 internal constant OPTIMISM_CHAIN_ID = 10;
+
+  // Deploy parameters
+  // Chainlink VRF
+  uint32 internal constant CHAINLINK_CALLBACK_GAS_LIMIT = 1_000_000;
```

Comment:
Seems high....1m gas on ETH would be very expensive.  @PierrickGT @trmid What would a realistic prod number look like?

Reply:
We can use this tool to estimate the gas limit: https://docs.chain.link/vrf/v2/estimating-costs
Some more docs here to figure out the value: 
https://docs.chain.link/vrf/v2/direct-funding/supported-networks
https://docs.chain.link/vrf/v2/direct-funding/#limits

Reply:
Trevor said 200_000k gas is good